### ISSUE TITLE
Highlighting the clicked word

### ISSUE THREAD

**Issue by drdhaval2785** [mbykov/morpheus#3]
Wouldn't it be a good idea to keep the clicked word highlighted so that it is easily found out.
In long prose, the user would like to see the word highlighted, otherwise it takes 3-4 seconds to search for the word whose details are given in popup box

**Comment by drdhaval2785** [comment]
@mbykov any response?

**Comment by mbykov** [comment]
yes, it is a good idea. I think about highlighting clicked word, and may be some color scheme - verbs are green, nouns are red, etc. It has to be a separate module in code with the possibility of some tuning.

**Comment by drdhaval2785** [comment]
@mbykov 
Ding Dong

**Comment by drdhaval2785** [comment]
http://www.the-art-of-web.com/javascript/search-highlight/ is the best deal I can see in a cursory search.
javascript.

@mbykov

**Comment by mbykov** [comment]
2015-12-09 11:55 GMT+03:00 Dr. Dhaval Patel <notifications@github.com>:

> http://www.the-art-of-web.com/javascript/search-highlight/ is the best
> deal I can see in a cursory search.
> javascript.
>
> @mbykov <https://github.com/mbykov>
>
>
But look also https://github.com/timoxley/element-selector, there is demo. In
addition, this code can be used as a component, it is ready to use out of
the box.







> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mbykov/morpheus/issues/3#issuecomment-163153759>.
>



-- 
М.

http://diglossa.ru
xmpp://m.bykov@jabber.ru

**Comment by drdhaval2785** [comment]
Whatever you think fit. Just do it.
:+1:
